docs/index.md

```diff
@@ -187,58 +187,62 @@ React Native Firebase cannot be used in the "Expo Go" app, because [it requires
 
 ### Expo Bare Workflow
```

Comment:
I'd suggest moving this section after the next section (at the end of the Expo section) since this causes confusion sometimes as to which workflow/path to pick Bare workflow vs Development builds (whether locally or with EAS).

Reply:
I would like to keep this at the top of the section.  It is important that someone recognize this distinction early in their learning/reading.  We have all run into people trying to use RNF with Expo Go and thinking that they've gotten quite far when in fact they don't understand this fundamental fact (I just had a back-and-forth on this yesterday on [Firebase Me](https://firebase.me/) ).

Reply:
I agree with all of that and I'd leave this up to you. But to reduce the amount of confusion around "Bare" vs. "Expo Go" vs. "development build," &mdash; one way I can think of that is to make development builds the guided path by providing the reason why it doesn't work with Expo Go. This should satisfy most use cases and also provide confidence ot the reader that yes, this is the path they should pick when starting a new project. Also, with most bare workflow projects and projects that use prebuild, there is little distinction left between the two workflows.

Reply:
I'm cool to drop "Managed" in favour of "development build" (though, TBH, I find that term to be incorrect/misleading too....these builds aren't "development builds" they are "custom builds" or "configured builds" or something along those lines...they aren't only for development purposes).